generic/tests/netperf.py

```diff
@@ -74,13 +74,35 @@ def mtu_set(mtu):
         if netdst in br_in_use:
             ifaces_in_use = host_bridges.list_iface()
             target_ifaces = list(ifaces_in_use + br_in_use)
-        if params.get("netdst_nic1") in process.system_output(
-                "ovs-vsctl list-br", ignore_status=True, shell=True).decode():
-            ovs_list = "ovs-vsctl list-ports %s" % params["netdst_nic1"]
-            ovs_port = process.system_output(ovs_list,
-                                             shell=True).decode().splitlines()
-            target_ifaces = target_ifaces + \
-                params.objects("netdst_nic1") + ovs_port
+        if process.run("which ovs-vsctl && "
```

Comment:
`process.run` will return a result object, not the exit code, and `ignore_status=False`, so when the command is failed, the test will break directly.

Reply:
I have updated, please help review.